### ISSUE TITLE
CELA P1_A11y_code.visualstudio.com_VisualStudioCode_October 2018 version 1.29(Workbench)_Screenreader : Screenreader is not announcing brackets,colon,etc symbols in the code section.

### ISSUE THREAD

**Issue by pranshu512** [microsoft/vscode-docs#3604]
[“Check out Accessibility Insights!](https://accessibilityinsights.io/) - Identify accessibility bugs before check-in and make bug fixing faster and easier.”

**GitHubTags**:#CELAP1_A11y_code.visualstudio.com;#A11ySev2;#A11yTCS;#A11yMAS;#Benchmark;#ChromiumEdge;#DesktopWeb;#MAS1.3.1;#Narrator;#Win10;#Visual Studio Code;#CELAP1_code;#Chrome;#NVDA

**Environment Details:**
Application Name: code.visualstudio.com
#URL: https://code.visualstudio.com/updates/v1_29
Windows Version: Windows10
Browser version: Microsoft Edge Version 84.0.488.1 (Official build) dev (64-bit)

**Repro Steps:**
1)Launch the application url :
https://code.visualstudio.com/updates/v1_29
2)October 2018 (version 1.29) Page will get open.
3)Navigate to code section under "Workbench navigation commands".
4)Launch screen reader and check whether it announce brackets,colon,etc symbols in the code section or not.

**Actual:**
Screen reader is announcing only textual content in the code section and not announcing brackets,colon,etc symbols.

**Same issue has been observed for all the brackets,colon,etc symbols present inside the code section in throughout application.**

**Same issue also observed with NVDA+Chrome.**

**Expected:**
Screenreader should announce brackets,colon,etc symbols in the code section along with textual content.

**User Impact:**
Screen reader users will not understand the code properly.

**Attachment link for Reference:**
[3604_CELA P1_A11y_code.visualstudio.com_VisualStudioCode_October 2018 version 1.29(Workbench)_Screenreader.zip](https://github.com/microsoft/vscode-docs/files/4528756/3604_CELA.P1_A11y_code.visualstudio.com_VisualStudioCode_October.2018.version.1.29.Workbench._Screenreader.zip)

**Comment by gregvanl** [comment]
This is an issue with the shiki syntax highlighter https://github.com/octref/shiki

See https://github.com/octref/shiki/issues/39

Marking as External

**Comment by Tyriar** [comment]
It looks to me like there is nothing wrong with the markup outlined in https://github.com/octref/shiki/issues/39. It's even semantically correct in that it's inside a `pre` and a `code` block, if a screen reader isn't reading it correctly it's because the screen reader isn't good at reading code. I think this should be closed as designed.

**Comment by gregvanl** [comment]
A VS Code engineer has reviewed this issue and believes it is a screen reader issue.

**Comment by octref** [comment]
Thanks @Tyriar, I'll close my upstream issue as well then.

**Comment by PoojaNamde** [comment]
@robole We are not able to find similar control in the https://code.visualstudio.com/api, could you please help

**Comment by Rama8345** [comment]
Verified the issue at https://code.visualstudio.com/api/get-started/extension-anatomy the issue is fixed. so, closing the bug. Please find attachment for reference.
**Github tags:**#Closed;

![3604_Closed](https://user-images.githubusercontent.com/115216656/228565427-ebbb39b1-55fe-4204-b090-da0b4f873b88.gif)
